Taskfile.yml

```diff
@@ -162,8 +162,7 @@ tasks:
         {{ range $dep := (.brew_dependencies | trim | splitLines) -}}
           [ -n "$(which {{ $dep }})" ]
         {{ end -}}
-      - |
-        [ "$(npm -v | awk -F. '{print ($1 > 9 || ($1 == 9 && $2 > 4)) ? 0 : 1}')" -eq 0 ]
+      - "[ $(npm -v | cut -d. -f1) -ge 9 ]"
```

Comment:
It looks like this merge didn't go as intended — this should not be changed in this PR

Reply:
Yes - I realized that I had missed these. The latest commit contains those lines